docs/workspace.dsl

```diff
@@ -26,8 +26,8 @@ workspace "${SERVICE_NAME}" "Mozilla's service providing email and phone masks."
                 description "The Mozilla Telemetry Platform"
                 tags "Other Software System"
             }
-            metrics_system = softwareSystem "Operational Metrics Platform" {
-                description "The Mozilla operational metrics and reporting platform"
+            metrics_system = softwareSystem "Google Managed Prometheus" {
+                description "Stores metrics for queries and reports"
```

Comment:
question (non-blocking): E.g., Do we need to change this label from "Operational Metrics Platform", "The Mozilla operational metrics and reporting platform" to the specific "Google Managed Prometheus" label? Insofar as someone needs to understand the Relay architecture, do they need to know that Mozilla's operational metrics & reporting platform is Google Managed Prometheus? Or can we keep this as the more generic "Operational Metrics Platform" and link it to another document that has the technical details?